src/components/Search.vue

```diff
@@ -1,48 +0,0 @@
-<template>
-  <div class="row">
-    <div class="col-sm-12 col-md-6 form-group">
-      <label for="searchInputName">Search by name:</label>
-      <input
-        type="text"
-        class="form-control"
-        id="searchInputName"
-        placeholder="Name"
-        name="filterName"
-        @change="onChange($event)"
-      />
-    </div>
-    <div class="col-sm-12 col-md-6 form-group">
-      <label for="sortInputId">Sort By Id:</label>
-      <select
```

Comment:
same as above

Reply:
Since this was VueJS Training assignment and not Accessibility training, I haven't paid much attention to sr or aria tags used while implementing accessibility.